docs/python/coding-pack-python.md

```diff
@@ -0,0 +1,145 @@
+
+## Getting started
+
+With the Coding Pack for Python it's easy to get started developing with Python and VS Code.
+
+1. [Download](https://aka.ms/coding-pack-for-python-win) and run the Coding Pack for Python installer.
+   >**Note**: The installer only supports installation on 64-bit Windows.
+1. Once the installer launches, review and accept the License Agreement. Then select **Install**.
+
+   ![Coding Pack for Python start installation screen](images/coding-pack/coding-pack-python-install-01.png)
+
+1. After installation completes, select **Next**.
+
+   ![Coding Pack for Python installation progress screen](images/coding-pack/coding-pack-python-install-02.png)
+
+   >**Note**: If you select **Cancel** before installation completes, you will need to manually remove and uninstall any components that have already been installed.
+
+1. Launch Visual Studio Code and start coding!
+
+   ![Coding Pack for Python installation complete screen](images/coding-pack/coding-pack-python-install-03.png)
+
+>**Note**: If there are any issues installing components, you can use the steps discussed in [Manual installation](#manual-installation)
+
+## Manual installation
+
+If you have any problems during installation, the following manual steps can be used to complete your installation.
+
+### Visual Studio Code and the Python extension
+
+1. If there was an issue installing VS Code, you can install it from [here](https://code.visualstudio.com/).
+1. Once VS Code is installed, you can install the [Python extension for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-python.python) from the Visual Studio Marketplace. For additional details about installing extensions, see [Extension Marketplace](https://code.visualstudio.com/docs/editor/extension-gallery). The Python extension is named **Python** and is published by Microsoft.
+
+      [![Python extension on Marketplace](images/coding-pack/python-extension-marketplace-install.jpg)](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+
+### Python interpreter
+
+1. If there was an issue installing the Python interpreter, you can install Python 3.8 from the [Microsoft Store](https://www.microsoft.com/p/python-38/9mssztt1n39l?activetab=pivot:overviewtab). Along with the Python extension, you need to install a Python interpreter for development with Python. There are other options for installing the Python interpreter, such as directly from [Python.org](https://www.python.org/downloads/), and which interpreter you use is dependent on your specific needs. If you use the Python.org version, you can typically use the **Download Python** button that appears first on the page to download the latest version. Just make sure to uncheck the "Install launcher for all users" box if you don't have admin access.
```

Comment:
```suggestion
1. If there was an issue installing the Python interpreter, you can install Python 3.8 from the [Microsoft Store](https://www.microsoft.com/store/apps/9MSSZTT1N39L?cid=vscode-docs). Along with the Python extension, you need to install a Python interpreter for development with Python. There are other options for installing the Python interpreter, such as directly from [Python.org](https://www.python.org/downloads/), and which interpreter you use is dependent on your specific needs. If you use the Python.org version, you can typically use the **Download Python** button that appears first on the page to download the latest version. Just make sure to uncheck the "Install launcher for all users" box if you don't have admin access.
```

Updated the link to the Store page.

> If you use the Python.org version, you can typically use the **Download Python** button that appears first on the page to download the latest version.

Just a heads up that in about a month this button will change to Python 3.9. We might be better linking to https://www.python.org/downloads/release/python-385/

Reply:
Changed in the latest push.